torch/lib/THNN/generic/SpatialConvolutionMM.c

```diff
@@ -371,7 +371,7 @@ void THNN_(SpatialConvolutionMM_accGradParameters)(
 
   THTensor_(free)(input);
   THTensor_(free)(gradOutput);
-  THTensor_(free)(gradWeight);
+  THTensor_(free)(gradWeight); // TODO: freeCopyTo gradWeight
```

Comment:
actually, i'll be removing the TODO. We treat these buffers across all of THNN/THCUNN as mutable (and resizable), so this does not matter.